pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/QueryGenerator.java

```diff
@@ -1005,6 +1005,16 @@ public QueryFragment generatePredicate(String columnName, boolean useMultistageE
       List<String> columnValues = _columnToValueList.get(columnName);
       String leftValue = pickRandom(columnValues);
       String rightValue = pickRandom(columnValues);
+
+      if (_singleValueNumericalColumnNames.contains(columnName)) {
```

Comment:
Did you add this in order for the test to pass? We probably want to test scenarios when lower is larger than higher (always false scenario)